infra/base-images/base-runner/install_javascript.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -eux
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+# Install Node.js v19.x
+apt-get install -y curl
+
+curl -fsSL https://deb.nodesource.com/setup_19.x | bash -
+apt-get install -y nodejs
+
+# Install latest versions of npm and yarn
+npm install --global npm yarn
```

Comment:
Is yarn strictly needed?

Reply:
Let's remove then, in the interests of keeping our images leaner. 

Also, re the builder vs the runner images: Is `npm` required in the runner as well if we bundle Jazzer.js in the build?

Reply:
I've removed `npm` from the runner and updated the wrapper script to start the Jazzer.js CLI directly without `npx`.